platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/ClassPageRenderer.java

```diff
@@ -113,9 +113,29 @@ protected void renderFailures(SimpleHtmlWriter htmlWriter) throws IOException {
         }
     }
 
+    @Override
+    protected void renderIgnoredTests(SimpleHtmlWriter htmlWriter) throws IOException {
+        for (TestResult test : getResults().getIgnored()) {
+            htmlWriter.startElement("div").attribute("class", "test")
+                .startElement("a").attribute("name", test.getId().toString()).characters("").endElement() //browsers dont understand <a name="..."/>
+                .startElement("h3").attribute("class", test.getStatusClass()).characters(test.getDisplayName()).endElement();
+            for (SerializableFailure failure : test.getFailures()) {
+                String message;
+                if (GUtil.isTrue(failure.getMessage()) && !failure.getStackTrace().contains(failure.getMessage())) {
+                    message = failure.getMessage() + SystemProperties.getInstance().getLineSeparator() + SystemProperties.getInstance().getLineSeparator() + failure.getStackTrace();
+                } else {
+                    message = failure.getStackTrace();
+                }
+                codePanelRenderer.render(message, htmlWriter);
+            }
+            htmlWriter.endElement();
+        }
+    }
+
     @Override
     protected void registerTabs() {
         addFailuresTab();
+        addIgnoredTab();
```

Comment:
🤔 It is worth distinguishing ignored from skipped (due to assumption)?

There might be very different use cases for these.  I can imaging wanting to find tests affected by either one or the other mechanism.

Reply:
Maybe we should call this page "Skipped" instead of Ignored because it's never only included `@Ignore` tests. It's always been the list of tests that were skipped.  We seem to use the word "ignored" in these reports to mean "skipped".

With this change, the list of tests on the "Ignored tab" will come in two flavors:
- The test name and no other details. This means the test was ignored. It never started execution.
- The test name and the assumption failure. This means the test was skipped. It started execution and was aborted.

I'm wary to try to improve these pages as-is because I think a bigger revamp would be a better use of time.

Reply:
Skipped is much better than Ignored already, if we don't want to make other changes here.